.yarn/patches/@types-mocha-npm-10.0.1-7c94e9e170.patch

```diff
@@ -0,0 +1,156 @@
+diff --git a/index.d.ts b/index.d.ts
```

Comment:
The Mocha types clash with Jest's types resulting in type errors, so I removed all Mocha globals. They can be imported from `mocha` instead, if desired:

```ts
import { describe, it, /* ... */ } from 'mocha';
```

Reply:
Is there a way to configure this instead of patching? 🤔

Reply:
I don't think so, since all `@types/*` stuff is loaded by default.

Reply:
Please add a comment in the patch explaining the problem and that the dev now needs to import manually